tck/src/main/java/org/eclipse/microprofile/lra/tck/TckTestBase.java

```diff
@@ -124,15 +112,15 @@ String checkStatusReadAndCloseResponse(Response.Status expectedStatus, Response
      * The started LRA will be named based on the class name and the running test name.
      */
     String lraClientId() {
-        return this.getClass().getSimpleName() + "#" + testName.getMethodName();
+        return lraTestService.lraClientId(this.getClass(), testName.getMethodName());
```

Comment:
You have moved a one line statement into another class and it is only ever called once. Can we keep the original statement.